docs/docs.syscalls.js

```diff
@@ -114,8 +114,8 @@ export default {
     'api/validation/locationValidator': function (_, payload) {
       console.log("Hit VML and validate!");
     },
-    'api/validation/emailDNCValidator': function(_, payload) {
-      console.log('Hit VML and check if email is on DNC list');
+    'api/validation/emailDNCValidator': function (_, payload) {
+      console.log("Hit VML and validate!");
```

Comment:
I removed the Axios call because I thought we wanted to keep that out of production for RA. This file just contains documentation of the routes we are using, at least in RA, without making any calls. I can add it back if that's what we want to do.

Reply:
You do want the Axios call here so that you can test the validation within Periodic. The `docs` section isn't part of the build that's imported into RA, so it's okay.